.github/workflows/process-images.yml

```diff
@@ -13,7 +13,7 @@ jobs:
         with:
           ref: ${{ github.event.pull_request.head.sha }}
       - name: Tinify Image Action
-        uses: namoscato/action-tinify@v1.3.1
+        uses: namoscato/action-tinify@v1.2.6
```

Comment:
Is that really a desired change in this PR?

Reply:
@triclambert last night was checking why the image compress action is failing.